include/zephyr/net/wifi.h

```diff
@@ -30,6 +30,21 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
+/**
+ * @brief - Wi-Fi rate flags
+ */
+enum wifi_rate_flags {
```

Comment:
If this is only used for TX injection, then we should prefix it with `tx_injection`